pkg/rebuild/cratesio/infer.go

```diff
@@ -239,6 +239,14 @@ func (Rebuilder) InferStrategy(ctx context.Context, t rebuild.Target, mux rebuil
 			return nil, errors.New("rust version heuristic failed")
 		}
 	}
+	// TODO: This should be moved to build-time since strategies are intended to be, at least notionally, arch-independent.
+	hasMUSLBuild, err := reg.HasMUSLBuild(rustVersion)
+	if err != nil {
+		return nil, errors.New("rust version heuristic failed")
```

Comment:
how does one debug off the error in 245? its the same error as 239. Should you add the error returned from HasMUSLBuild in here? Should it say "while checking HasMUSLBuild"?